web/ui-go/Makefile

```diff
@@ -0,0 +1,19 @@
+PROTO_DIR := ../../core/lib/src/main/java/dev/enola/core
```

Comment:
Is it possible to replace this with https://github.com/bazelbuild/rules_go and avoid having a `Makefile` at all?